presto-main/src/test/java/io/prestosql/operator/scalar/TestMathFunctions.java

```diff
@@ -688,11 +688,14 @@ public void testRandom()
         functionAssertions.tryEvaluateWithAll("rand()", DOUBLE, TEST_SESSION);
         functionAssertions.tryEvaluateWithAll("random()", DOUBLE, TEST_SESSION);
         functionAssertions.tryEvaluateWithAll("rand(1000)", INTEGER, TEST_SESSION);
+        functionAssertions.tryEvaluateWithAll("random(1000, 2000)", INTEGER, TEST_SESSION);
         functionAssertions.tryEvaluateWithAll("random(2000)", INTEGER, TEST_SESSION);
         functionAssertions.tryEvaluateWithAll("random(3000000000)", BIGINT, TEST_SESSION);
+        functionAssertions.tryEvaluateWithAll("random(3000000000, 5000000000)", BIGINT, TEST_SESSION);
```

Comment:
Could you add more test cases? 
- tinyint
- smallint
- negative arguments
- both arguments are the same value